### ISSUE TITLE
Squamous metaplasia

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10073]
**Preferred term label:**
Squamous metaplasia

**Synonyms**

**Definition (free text, please give PubMed ID)**
https://www.ncbi.nlm.nih.gov/medgen/C0025570

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
 Intraductal papilloma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
I am moving forward with a new histopathology section of the HPO. The intended use case would be within a biospecimen section of the Phenopacket (or similar other data structures).

Adding

HP_0020094
Squamous metaplasia
DEF: Metaplasia of an epithelium (normally a unilayer of cuboidal or columnar cells) with squamous epithelium (squamous means scale-like). Metaplastic squamous epithelium becomes stratified (i.e., has multiple layers).
